pynucastro/networks/python_network.py

```diff
@@ -193,6 +196,21 @@ def _write_network(self, outfile=None):
         for n in self.unique_nuclei:
             of.write(f"Z[j{n}] = {n.Z}\n")
 
+        # we'll compute the masses here in erg
+
+        m_n_MeV = constants.value('neutron mass energy equivalent in MeV')
+        m_p_MeV = constants.value('proton mass energy equivalent in MeV')
+        m_e_MeV = constants.value('electron mass energy equivalent in MeV')
+        MeV2erg = (constants.eV * constants.mega) / constants.erg
+
+        of.write("\n")
+
+        of.write("# masses in ergs\n")
+        of.write("mass = np.zeros((nnuc), dtype=np.float64)\n\n")
+        for n in self.unique_nuclei:
+            mass = ((n.A - n.Z) * m_n_MeV + n.Z * (m_p_MeV + m_e_MeV) - n.A * n.nucbind) * MeV2erg
```

Comment:
It might be cleaner to pull this into a method on `Nucleus` so we're not duplicating the code in `RateCollection.evaluate_energy_generation()`

Reply:
yes, that's a good idea.  I'll move that this afternoon

Reply:
interesting... in `Nucleus` we already have `A_nuc`, which comes from the table.  So maybe we are being redundant when we recompute the mass?